data/strings/brands_strings.txt

```diff
@@ -38,15 +38,16 @@
     en = Blue Bottle Coffee
     ja = ブルーボトルコーヒー
     ko = 블루버틀 커피
-    zh = 藍瓶咖啡
+    zh-Hant = 藍瓶咖啡
```

Comment:
`zh-Hans` version should also be added if you remove a generic `zh` translation.

Reply:
add already